sample.env

```diff
@@ -1,3 +1,3 @@
 NODE_ENV=development
 PORT=3001
-dbUri=mongodb://localhost:27017/cars
+MONGODB_URI='mongodb://127.0.0.1:27017/cars'
```

Comment:
I don't think the quotes are required. I assume it doesn't cause a problem.

Reply:
does not matter